package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@netzstrategen/twig-asset",
+  "version": "1.0.0",
+  "description": "",
+  "main": "js/functions.js",
+  "repository": {
```

Comment:
Is it necessary to specify repository after renaming the repo?

Reply:
Yes, should be specified as npmjs.com is using it's information.